algorithms/symmetry/__init__.py

```diff
@@ -156,16 +156,27 @@ def _normalise(self, method):
         elif method == "ml_aniso":
             normalise = self.ml_aniso_normalisation
 
-        for i in range(int(flex.max(self.dataset_ids) + 1)):
-            logger.info("\n" + "-" * 80 + "\n")
-            logger.info("Normalising intensities for dataset %i\n" % (i + 1))
-            intensities = self.intensities.select(self.dataset_ids == i)
-            if i == 0:
-                normalised_intensities = normalise(intensities)
-            else:
-                normalised_intensities = normalised_intensities.concatenate(
-                    normalise(intensities)
-                )
+        try:
+            for i in range(int(flex.max(self.dataset_ids) + 1)):
+                logger.info("\n" + "-" * 80 + "\n")
+                logger.info("Normalising intensities for dataset %i\n" % (i + 1))
+                intensities = self.intensities.select(self.dataset_ids == i)
+                if i == 0:
+                    normalised_intensities = normalise(intensities)
+                else:
+                    normalised_intensities = normalised_intensities.concatenate(
+                        normalise(intensities)
+                    )
+        except RuntimeError:
+            logger.warning(
+                "A problem occurred when trying to normalise the intensities by the "
+                "%s method.\n"
+                "There may be too few unique reflections. Resorting to using "
+                "un-normalised intensities instead.",
+                method,
```

Comment:
Again, would it be useful to also report the specific error that was encountered when normalising the intensities?